_layouts/post.html

```diff
@@ -145,4 +145,4 @@
 You can submit an improvement to this page by
 <a href="https://github.com/endoflife-date/endoflife.date/blob/master/{{page.path}}"
 title="Click the Pencil, the link takes you directly to the correct page">visiting GitHub</a>.
-You will need to fork the website and submit a Pull Request.
+You will need to fork the website and submit a Pull Request. A JSON version of this page is available at <a href="/api/{{page.permalink}}.json">/api{{page.permalink}}.json</a>. See the <a href="https://github.com/endoflife-date/endoflife.date/wiki/API-Documentation">API Documentation</a> for more.
```

Comment:
For example:

https://deploy-preview-262--endoflife-date.netlify.app/django links to https://deploy-preview-262--endoflife-date.netlify.app/api//django.json

Replace the double `//` with a single `/` before `django.json`?